eventuate-core/src/it/scala/com/rbmhtechnology/eventuate/ReplicationIntegrationSpec.scala

```diff
@@ -43,18 +81,25 @@ object ReplicationIntegrationSpec {
     }
   }
 
+  class ManifestFilter(subString: String) extends ReplicationFilter {
```

Comment:
Do you plan adding this to main sources later (maybe extended with a regex-based matching)?